runtime/src/message_processor.rs

```diff
@@ -134,6 +135,10 @@ impl MessageProcessor {
                 timings,
             );
             time.stop();
+            if let Some(consumed_compute_units) = consumed_compute_units {
+                **consumed_compute_units =
+                    consumed_compute_units.saturating_add(compute_units_consumed);
```

Comment:
It's very confusing that `consumed_compute_units` and `compute_units_consumed` are two different things. Can one be `instruction_compute_units` and the other be `tx_compute_units`?